project/src/main/java/com/google/sps/servlets/VolunteeringFormHandlerServlet.java

```diff
@@ -14,34 +14,57 @@
 /** Servlet that handles submission of the form for creating volunteering opportunities. */
 @WebServlet("/volunteering-form-handler")
 public class VolunteeringFormHandlerServlet extends HttpServlet {
+  private static final String OPPORTUNITY_ID = "opportunity-id";
   private static final String OPPORTUNITY_NAME = "opportunity-name";
   private static final String OPPORTUNITY_NUM_SPOTS = "opportunity-num-spots";
   private static final String SKILL = "skill";
```

Comment:
For consistency, if this is used only in the context of volunteering opportunities, also name it OPPORTUNITY_X?

Reply:
Done - yes, I noticed now that my naming is a bit consistent. I have made it match all variables in the opportunity class across all files now.